python/GafferImageUI/ResampleUI.py

```diff
@@ -81,14 +81,16 @@
 
 		) ),
 
-		"filterWidth" : [
+		"filterScale" : [
 
 			"description",
 			"""
-			An override for the width of the filter used. This is
+			An override for the scale of the filter used. This is
```

Comment:
Hmm, yeah, this description is kind of failing.  The default means that the sampled area will effectively be one pixel wide *in output space".  But this doesn't mean that the it will be one pixel wide in input space, or that the filter support will be one pixel wide.  I feel like it's maybe worth mentioning that a cubic with a support of 2, or a box with a support of 1, or a lanczos3 with a support of 6, are all filters that are sampling an area of approximately 1 pixel, just with different sharpness.  But it seems like I've completely failed at communicating this, so maybe I should just remove a bunch of this comment.